test/controllers/responsiblepeople/TrainingControllerSpec.scala

```diff
@@ -173,7 +173,7 @@ class TrainingControllerSpec extends AmlsSpec with MockitoSugar with ScalaFuture
             redirectLocation(result) must be(Some(routes.DetailedAnswersController.get(recordId).url))
           }
 
-          "redirect to FitAndProperController when businessActivities includes TrustAndCompanyServices and phase 2 changes toggle is false" in new Fixture {
+          "redirect to FitAndProperNoticeController when businessActivities includes TrustAndCompanyServices and phase 2 changes toggle is false" in new Fixture {
```

Comment:
the test is modified to check the FitAndProperNoticeController is hit by the TrainingController.
Shouldnt we have a test on FitAndProperNoticeController to see it will hit the next controller i.e FitAndProperController

Reply:
Thanks @owen-opencast